addons/l10n_lu/__manifest__.py

```diff
@@ -7,7 +7,7 @@
 
 {
     'name': 'Luxembourg - Accounting',
-    'version': '2.0',
+    'version': '3.0',
```

Comment:
2.1 would be more logical as this is not a complete rework of the version, it is only a minor change (reports are the same, taxes are the same except for the amount,...)